### ISSUE TITLE
Can resave MLTSHP images to itself

### ISSUE THREAD

**Issue by shfitz** [MLTSHP/mltshp_plugins#1]
At least with the firefox plugin, it's possible to re-save an already existing image to the SHP. On MLKSHK, this duping wasn't permitted. See http://new.mltshp.com/p/PW and http://new.mltshp.com/p/PY for a dupe example.

**Comment by shfitz** [comment]
Related(?) it's not possible to save from mlkshk.com to mltshp.com.

**Comment by iLama** [comment]
I'd need more information but this is probably an issue with the site and not the plugins.

**Comment by shfitz** [comment]
Yeah, I don't see any reference to mlkshk in this. I'll poke around the site and see what I can find.

**Comment by ardgedee** [comment]
@shfitz Does this behavior still exist? Is it relevant to modern or legacy Firefox?

**Comment by shfitz** [comment]
@ardgedee It's still the case on Chrome (https://mltshp.com/p/1CVE5 was saved from https://mltshp.com/p/1CVE1) and Firefox (http://mltshp.com/p/1CVE6 was saved from https://mltshp.com/p/1CVE1)

**Comment by iLama** [comment]
To be clear, you are stating the issue is that the MLTSHP extension can be used to save images already on MLTSHP to MLTSHP as a new post? 

If this is the case, then this isn't really a bug with the extensions as much as it is with the main code base, and it might be better to report the bug on the main MLTSHP repo.

**Comment by ardgedee** [comment]
@iLama Hm... I think the point is mostly that an extension should be aware of when it's being used to re-save a mltshp post and do something else instead (eg, prompt the user to click the "Save" button on the post instead).

I'm OK with leaving this ticket open to address that as a feature enhancement, but I agree it's not really a bug.

**Comment by iLama** [comment]
@ardgedee fair enough, I'll assign it to myself and work on getting the changes into the extensions.
